project/Dependencies.scala

```diff
@@ -23,7 +23,7 @@ object Dependencies {
   val scalaXmlVersion = "1.2.0"
   val lmVersion = "1.0.0"
   val configDirsVersion = "10"
-  val caseAppVersion = "1.2.0-faster-compile-time"
+  val caseAppVersion = "2.0.6"
```

Comment:
Should we also remove the custom shapeless?  I think both forks were added because of caseapp? I checked it out locally and removed shapeless, compilation time is faster than before, so I think we no longer have use for it. Both were originally introduced here: https://github.com/scalacenter/bloop/pull/509

Reply:
I hadn't noticed the custom shapeless… I just removed it too.

Reply:
Thanks! One less fork :+1: